.github/workflows/knative-verify.yaml

```diff
@@ -45,22 +41,16 @@ jobs:
 
     - name: Install Dependencies
       run: |
-        curl -L https://github.com/google/ko/releases/download/v0.6.0/ko_0.6.0_Linux_x86_64.tar.gz | tar xzf - ko
-        chmod +x ./ko
-        sudo mv ko /usr/local/bin
-        go get github.com/google/go-licenses
+        go install github.com/google/ko@latest
```

Comment:
Jason Hall has an action to install a release of `ko`.  Installing from source takes like 60s FYI.